packages/v4/src/content/design-guidelines/components/date-picker/date-picker.md

```diff
@@ -0,0 +1,143 @@
+
+**Behavior**
+When date pickers and time pickers are combined, each control’s behavior will remain the same. After users select the start date and time, the end date and time will be automatically set to the same hour of the next day.
+
+Depending on your use case, time pickers and date pickers can be arranged in two main ways: Horizontal alignment and vertical alignment (in this case labels are recommended).
+
+**Horizontal**
+
+<img src="./img/Time and Date picker horizontal.png" alt="Example of a horizontal date and time range selection" width="550"/>
+
+For horizontally arranged date and time pickers, connect each combined date and time picker with "to" so that users know which date defines which point of the date range. Always arrange each date and time picker in chronological order from left to right, with the start picker on the left and the end picker on the right.
+
+**Vertical with labels**
+
+<img src="./img/Time and Date picker vertical with labels.png" alt="Example of a vertical date and time range selection with labels" width="242"/>
+
+For vertically arranged date and time pickers, always include field labels like "Starts" and "Ends" to indicate which picker corresponds to which point of the date range. Always arrange each date and time picker in chronological order from top to bottom, with the start picker on the top and the end picker below.
+
+#### ‘All day’ checkbox
+An ‘All day’ checkbox disables time pickers by indicating a user’s selection includes all hours of their selected date. Consider using an ‘All day’ checkbox to simplify combined date and time pickers for combined use cases where users only need to specify dates, or would like to select an entire day for an event.
+
+<img src="./img/Picker with All day checkbox.png" alt="Example of a vertical date and time range selection with 'All day' checkbox" width="680"/>
+
+## Variations
+
+### Toolbar
+Use date pickers in a toolbar to filter dates, mainly as past timestamps.
+
+#### In toolbar attribute filter
+Date range fields in an attribute filter appear after selecting the Date range filter option which will push other content in the toolbar (in this case CTA button) to the right. The date(s) selected should display as chips under the filter.
+
+<img src="./img/In toolbar attribute filter.png" alt="Example of date picker in toolbar attribute filter" width="500"/>
+
+#### In toolbar filter group
+Use a date picker as its own date-range filter in a toolbar filter group when date-range filtering is a common action.
+Since horizontal space is limited in toolbar filter groups, use placeholder text to designate the range’s start and end dates.
+
+For more guidance on using filter groups, see our [toolbar design guidelines](/components/toolbar/design-guidelines).
+
+<img src="./img/Date picker in toolbar.png" alt="Example of date picker in toolbar filter group" width="780"/>
```

Comment:
Can you change this image so that it's not using an attribute filter again? Maybe just have the search input for the name, and get rid of the "Name" dropdown in front so that it just becomes a regular search.